runtime/src/accounts_db.rs

```diff
@@ -3254,7 +3254,7 @@ impl AccountsDb {
         let mut hasher = Hasher::default();
 
         hasher.hash(&account.data());
-        hasher.hash(&account.owner.as_ref());
+        hasher.hash(&account.owner().as_ref());
```

Comment:
Doesn't `.owner()` already return a ref?

Reply:
as_ref on the pubkey causes it to return a &[u8], which is what hasher wants.
The leading & appears to be unnecessary but acceptable.

Reply:
in pubkey.rs:
impl AsRef<[u8]> for Pubkey {
    fn as_ref(&self) -> &[u8] {
        &self.0[..]
    }
}

Reply:
This was a great question, though.

Reply:
Gotcha, thanks!